tests/test_codebase/test_mmocr/test_mmocr_models.py

```diff
@@ -506,7 +503,7 @@ def test_sar_model(backend: Backend, decoder_type):
     pytorch_model.cfg = sar_cfg
     patched_model = patch_model(
         pytorch_model, cfg=deploy_cfg, backend=backend.value)
-    onnx_file_path = tempfile.NamedTemporaryFile(suffix='.onnx').name
+    onnx_file_path = str(tmp_path / 'tmp.onnx')
```

Comment:
does this work for windows?

Reply:
[tmp_path](https://docs.pytest.org/en/7.2.x/reference/reference.html#tmp-path) is a builtin fixture in pytest which return a [pathlib.Path](https://docs.python.org/3/library/pathlib.html#pathlib.Path) object.